bin/test

```diff
@@ -58,16 +58,17 @@ if [ "$coverage_flags" != '' ]; then
     mkdir -p .coverage
     # Compile the .gcdo files into an intermediate format.
     # Use the faster JSON Perl module, JSON:XS, to speed things up.
-    lcov --quiet --capture --directory .build --output-file .coverage/raw.info --rc geninfo_json_module=JSON::XS
+    lcov --quiet --capture --directory .build --output-file .coverage/raw.info #--rc geninfo_json_module=JSON::XS
```

Comment:
```suggestion
    lcov --quiet --capture --directory .build --output-file .coverage/raw.info --rc geninfo_json_module=JSON::XS
```

Reply:
This is probably exactly why you commented it: to try it out. Did you find where `geninfo_json_module` is defined? I couldn't find any info

Reply:
It is working but codecode would not consume `.coverage/raw.info`, I thought maybe codecov is struggling to parse the JSON output.